src/test/java/core/JdkTest.java

```diff
@@ -12,14 +12,14 @@
 
     @Test @TestActivation({"ORACLE_LOGIN", "ORACLE_PASSWORD"})
     public void autoinstallJdk() {
-        final String login = System.getenv("ORACLE_LOGIN");
-        final String passwd = System.getenv("ORACLE_PASSWORD");
+        final String login = System.getProperty("JdkTest.ORACLE_LOGIN");
+        final String passwd = System.getProperty("JdkTest.ORACLE_PASSWORD");
 
         ToolInstallation.waitForUpdates(jenkins, JdkInstallation.class);
 
         JdkInstallation jdk = ToolInstallation.addTool(jenkins, JdkInstallation.class);
-        jdk.name.set("jdk_1.7.0");
-        jdk.installVersion("jdk-7u11-oth-JPR");
+        jdk.name.set("jdk_1.8.0");
+        jdk.installVersion("jdk-8u141-oth-JPR");
```

Comment:
Updating to use most recent release that requires a login. Also, `jdk-7u11-oth-JPR` fails on OS X, seems to be a problem with the `.dmg`.